app/views/posts/show.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>Posts#show</h1>
-<p>Here is a list of posts for a given user</p>
+
+  <div class="flex flex-col">
+  <div class="flex justify-between w-full">
+    <h2 class="flex text-bold text-2xl"><%= @post.title%></h2>
+    <ul class="flex justify-between w-48">
+      <li>Comments: <%= @post.comments_counter%></li>
+      <li>Likes: <%= @post.likes_counter%></li>
+    </ul>
+  </div>
```

Comment:
- [ ] Kindly make sure your Post show page displays post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png) Here is your page 👇🏻 

![Screenshot from 2023-01-24 20-38-32](https://user-images.githubusercontent.com/79058364/214379729-248ac762-14cc-4aeb-a9b8-8990193e8388.png)